docs/operator-docs/update-cumulus-dependencies.md

```diff
@@ -0,0 +1,70 @@
+---
+id: update-cumulus-dependencies
+title: Update Cumulus Dependencies
+hide_title: true
+---
+
+# Updating Cumulus Dependencies
+
+This document will outline, step by step, how to update a Cumulus deployment to a newer version. It is currently assumed that _all_ Cumulus packages are to be updated to the same version at the same time. This is expected to change in the future.
```

Comment:
I would take out `this is expected to change...` We'll update the docs when it changes.